Makefile

```diff
@@ -14,6 +15,7 @@ E2E_TEST_EXTRA_ARGS ?=
 E2E_TEST_ARGS ?= $(strip -test.v -ginkgo.v $(E2E_TEST_EXTRA_ARGS))
 
 OPERATOR_SDK ?= build/_output/bin/operator-sdk
+GITHUB_RELEASE ?= build/_output/bin/github-release
```

Comment:
there are several groups of variable definitions, why is the one with OPERATOR_SDK and LOCAL_REGISTRY the best fit for GITHUB_RELEASE? I think we are starting to have a junkyard of variables here. It's getting out of hand, we should organize and comment them properly, in a separate PR.

Reply:
Yep will organize, in a other PR.